.github/renovate.json5

```
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval",
+        ":semanticCommitScopeDisabled"
+    ],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait a few days for any new package, as a precaution against malicious publishes
+    "minimumReleaseAge": "3 days",
+
+    "packageRules": [
+        // Enable major version updates, but don't auto-merge them
+        {
+            "automerge": false,
+            "matchUpdateTypes": ["minor"]
```

Should this be "major"?
```suggestion
            "matchUpdateTypes": ["major"]
```